### PR TITLE
Fix unpack issues

### PR DESCRIPTION
**Pull request by hanchuntao**
[OLCS-249](https://issues.redhat.com/browse/OLCS-249)

Fixes OLCS-249

Closes OLCS-249

### CODE REVIEW

**Comment by hanchuntao** [comment]
@rhyw @qduanmu @huiwangredhat PTAL

File: openlcs/libs/sc_handler.py [hunk 1]
```diff
@@ -113,3 +123,69 @@ def get_container_components(self, srpm_dir, misc_dir, sc_nvr):
...
+        source_dir = os.path.join(self.dest_dir, "blobs", "sha256")
+        srpm_link_dir = os.path.join(self.dest_dir, 'rpm_dir')
+        if os.path.exists(srpm_link_dir):
+            srpm_names = os.listdir(srpm_link_dir)
+            for srpm_name in srpm_names:
+                try:
+                    soft_link_path = os.path.join(srpm_link_dir, srpm_name)
+                    sha256_str = os.path.basename(os.readlink(soft_link_path))
+                    sha256_path = os.path.join(source_dir, sha256_str)
+                    dest_path = os.path.join(srpm_link_dir, srpm_name)
```
Comment (huiwangredhat) [inline openlcs/libs/sc_handler.py:140]: The dest_path is the same as the soft_link_path? If so what's the meaning of the dest_path?

File: openlcs/libs/common.py [hunk 2]
```diff
@@ -10,6 +23,22 @@ def create_dir(directory):
         if os.path.exists(directory):
             shutil.rmtree(directory, ignore_errors=True)
         os.makedirs(directory)
-    except Exception as e:
-        raise RuntimeError(e) from None
+    except Exception as err:
+        raise RuntimeError(err) from None
```
[the remark this answers is not in the document]
Comment (hanchuntao) [reply openlcs/libs/common.py:28]: No difference, just for unification, most of code use err

File: openlcs/libs/sc_handler.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        self.config = config
+        self.src_file = src_file
+        self.dest_dir = dest_dir
+
+    def collate_source_container_srpm(self):
+        """
+        Unpack the source RPM in source container.
+        """
+        source_dir = os.path.join(self.dest_dir, "blobs", "sha256")
+        srpm_link_dir = os.path.join(self.dest_dir, 'rpm_dir')
```
[the remark this answers is not in the document]
Comment (hanchuntao) [reply openlcs/libs/sc_handler.py]: Updated, use `soft_link_dir ` is more meaningful

File: openlcs/libs/sc_handler.py (hunk 1 above)
Comment (hanchuntao) [reply openlcs/libs/sc_handler.py]: Yes, this is a mistake, removed one of them, only use soft_link_path

**Comment by huiwangredhat** [comment]
LGTM.

**Pull request merged**